buildSrc/src/main/java/NativeLibsPlugin.kt

```diff
@@ -6,10 +6,10 @@
  * This gradle plugin generates a `copyNativeLibs` task in the project it is
  * applied to.
  * This task copies library files in the application-services libs/ folder
- * inside the build dir nativeLibs/ folder and makes sure this folder is
- * included in Android jniLibs sourcesets.
- * It is the responsability of the consumer of this plugin to depend on that
- * newly defined task, for example in the `generateDebugAssets` task.
+ * inside the build dir nativeLibs/ folder.
+ * It is the responsability of the consumer of this plugin to add `nativeLibs`
```

Comment:
Consumer here means us still, right? Not lockbox/r-b/fenix/a-c

Reply:
Yup!